src/main/scala/com/codacy/client/stash/util/JsonEnumeration.scala

```diff
@@ -0,0 +1,13 @@
+package com.codacy.client.stash.util
+
+import play.api.libs.json.Format
+
+trait JsonEnumeration extends Enumeration {
+  self: Enumeration =>
+
+  implicit lazy val format = Format(Implicits.enumReads(self), Implicits.enumWrites)
+
+  def findByName(name: String): Option[Value] = {
+    values.find(v => v.toString.toLowerCase == name.toLowerCase)
```

Comment:
it could've been name.equalsIgnoreCase(v.toString)

Reply:
I will fix that on another PR.